### PR TITLE
Backport membership integration

### PR DESCRIPTION
**Pull request by OleksandrRiumshyn**
Jira issue: [YGSTR-38](https://fivejars.atlassian.net/browse/YGSTR-38)

Changes during backport:
- Service override for importer replaced to query tag and alter
- Default category UUID reference replaced by ID reference and config ignore integration
- Updated documentation

### CODE REVIEW

File: modules/openy_traction_rec_membership/src/Commands/MembershipImportCommands.php [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+   *
+   * @command openy-tr-membership:fetch-all
+   * @aliases tr-membership:fetch
+   */
+  // phpcs:ignore
+  public function fetch(): void {
+    // Create additional command and run parent logic with new constructor.
+    parent::fetch();
+  }
+
```
Comment (anpolimus) [inline modules/openy_traction_rec_membership/src/Commands/MembershipImportCommands.php:87]: It looks like this file does not adds anything to the original file.

File: modules/openy_traction_rec_membership/src/MembershipImporter.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
```
Comment (anpolimus) [inline modules/openy_traction_rec_membership/src/MembershipImporter.php:3]: do we really need strict mode in a such simple file?

**Review by anpolimus — CHANGES_REQUESTED** [review]

**Comment by anpolimus** [comment]
Maybe lets move PR to the YUSA repo of this module?

File: modules/openy_traction_rec_membership/src/Commands/MembershipImportCommands.php (hunk 1 above)
Comment (OleksandrRiumshyn) [reply modules/openy_traction_rec_membership/src/Commands/MembershipImportCommands.php:87]: @anpolimus Yes, but it provides additional drush commands (see comments) and uses different services (see drush.services.yml)

File: modules/openy_traction_rec_membership/src/MembershipImporter.php (hunk 2 above)
Comment (OleksandrRiumshyn) [reply modules/openy_traction_rec_membership/src/MembershipImporter.php:3]: @anpolimus For now maybe not, but it just was done according to code sniffer recommendations and could be useful in future

**Comment by manachynskyi** [comment]
@anpolimus This module is currently in development, so we’ll share it, along with a few other features, once the related work is complete.

**Pull request merged**